### ISSUE TITLE
(Bug report) Blank shared notes don't show children

### ISSUE THREAD

**Issue by sigaloid** [zadam/trilium#2536]
### Trilium Version

0.49.3

### What operating system are you using?

Other Linux

### What is your setup?

Local + server sync

### Operating System Version

Fedora

### Description

Blank note:
![Screenshot from 2022-01-08 17-40-34](https://user-images.githubusercontent.com/69441971/148662377-76771c08-3d1e-4dc7-baf4-134ab455560c.png)
With test content: 
![Screenshot from 2022-01-08 17-40-55](https://user-images.githubusercontent.com/69441971/148662385-9f7255da-73f5-47c7-b40a-0ab81cd924d9.png)

Looks like https://github.com/sigaloid/trilium/blob/master/src/views/share/page.ejs#L52 fails on a blank note even though https://github.com/sigaloid/trilium/blob/master/src/views/share/page.ejs#L53 implies it should still pass that if statement.

TLDR: does `if (note.hasChildren()) {` fail for blank notes for some reason?

**Comment by zadam** [comment]
Thanks, it seems a [merge](https://github.com/zadam/trilium/commit/168645cce90a6c1fb1d56463da9aa5f2597b5fda) got it messed up somehow ...

**Comment by sigaloid** [comment]
:disappointed: reporting a bug I didn't know I caused... that's a new one. Sincere apologies, and thanks!
